pytests/tests/test_config.py

```diff
@@ -1,57 +1,62 @@
 #
-# Copyright (C) 2019 VMware, Inc. All Rights Reserved.
+# Copyright (C) 2019-2022 VMware, Inc. All Rights Reserved.
 #
 # Licensed under the GNU General Public License v2 (the "License");
 # you may not use this file except in compliance with the License. The terms
 # of the License are located in the COPYING file of this distribution.
 #
-#   Author: Siddharth Chandrasekaran <csiddharth@vmware.com>
 
 import os
 import pytest
 
+
 @pytest.fixture(scope='module', autouse=True)
 def setup_test(utils):
     tdnf_conf = os.path.join(utils.config['repo_path'], 'tdnf.conf')
     tdnf_repo = os.path.join(utils.tdnf_config.get('main', 'repodir'), 'photon-test.repo')
-    utils.run([ 'mkdir', '-p', '/tmp/myrepo' ])
-    utils.run([ 'cp', tdnf_conf, '/tmp/myrepo/mytdnf.conf' ])
-    utils.run([ 'sed', '-i', '/repodir/d', '/tmp/myrepo/mytdnf.conf' ])
-    utils.run([ 'sed', '-i', '$ a repodir=/tmp/myrepo', '/tmp/myrepo/mytdnf.conf' ])
-    utils.run([ 'cp', tdnf_repo, '/tmp/myrepo/photon.repo' ])
-    utils.run([ 'sed', '-i', 's/enabled=0/enabled=1/g', '/tmp/myrepo/photon.repo' ])
+    utils.run(['mkdir', '-p', '/tmp/myrepo'])
+    utils.run(['cp', tdnf_conf, '/tmp/myrepo/mytdnf.conf'])
+    utils.run(['sed', '-i', '/repodir/d', '/tmp/myrepo/mytdnf.conf'])
+    utils.run(['sed', '-i', '$ a repodir=/tmp/myrepo', '/tmp/myrepo/mytdnf.conf'])
+    utils.run(['cp', tdnf_repo, '/tmp/myrepo/photon.repo'])
+    utils.run(['sed', '-i', 's/enabled=0/enabled=1/g', '/tmp/myrepo/photon.repo'])
```

Comment:
Not a comment for this review directly, but I think some time we should remove the shell calls calls and use python instead.

Reply:
Agree but this is are simple test cases. Keeping them simple makes life easy. We need to write a bunch of functions to achieve this task but not a big deal TBF.